v-next.sql

```diff
@@ -1,3 +1,5 @@
+\i ocd-3307.sql
```

Comment:
You don't need to do this, or have the ocd-3307 file at all. If you're updating a view it is automatically replaced with line 4 of this file. View changes don't need anything in v-next or a specific file